packages/build-tools/resources/__eas/repack.yml

```
@@ -0,0 +1,12 @@
+build:
+  name: Repack golden app
+  steps:
+    - eas/checkout
+    - eas/install_node_modules
+    - eas/__download_and_repack_golden_development_client_archive:
+        id: download_and_repack_golden_development_client_archive
+    - eas/upload_artifact:
+        name: Upload build artifact
+        inputs:
+          type: application-archive
+          path: ${ steps.download_and_repack_golden_development_client_archive.repacked_archive_path }
```

```suggestion
          path: ${ steps.download_and_repack_golden_development_client_archive.repacked_archive_path }

```
newline at the end of file